spec.bs

```diff
@@ -4558,14 +4558,16 @@ dictionary StorageInterestGroup : AuctionAdInterestGroup {
 
 *This first introductory paragraph is non-normative.*
 
-{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an <dfn>auction nonce</dfn>, a
-one-time [=version 4 UUID=] uniquely associated with a single call to
-{{Window/navigator}}.{{Navigator/runAdAuction()}}. For multi-seller auctions, this ID is uniquely
-associated with all {{AuctionAdConfig/componentAuctions}}. This nonce will need to be passed back in
-via a subsequent call to {{Window/navigator}}.{{Navigator/runAdAuction()}} via the
-{{AuctionAdConfig}}. This is currently only needed for [=auctions=] that use [=additional bids=],
-for which the [=auction nonce=] will be included in each [=additional bid=] as a way of ensuring
-that those bids are only used in the [=auctions=] for which they were intended.
+{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an <dfn>auction nonce</dfn>, which
+is a one-time canonical [=string=] representation of a [=version 4 UUID=] that is uniquely
+associated with a single call to {{Window/navigator}}.{{Navigator/runAdAuction()}}. For multi-seller
+auctions, a (separate) auction nonce can be uniquely associated with each of the

```

Comment:
Perhaps s/(separate)/distinct/? (note, dropping the parenthesis)

Reply:
Done.